pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -699,10 +699,10 @@ type EmailConfig struct {
 	Headers []KeyValue `json:"headers,omitempty"`
 	// The HTML body of the email notification.
 	// +optional
-	HTML string `json:"html,omitempty"`
+	HTML *string `json:"html,omitempty"`
```

Comment:
Do still wanna make improvements and bug fixed on alpha apis?

Reply:
for now, v1alpha1 is still the default version being installed. I started #5398 a while ago to move forward with v1beta1 but no time to complete it for now...

Reply:
Got it, thx